internal/endpoints/endpointsFetcher_test.go

```diff
@@ -0,0 +1,12 @@
+package endpoints
+
+import "testing"
+
+func Test_GetListOfEndpoints(t *testing.T) {
+	var EXPECTED_ENDPOINTS = 354
+	var endpoints = GetListOfEndpoints("../../resources/EndpointSources.json")
```

Comment:
I had to change this to `../../../resources/EndpointSources.json` to get the tests to work, probably just a missed like after revamping the project structure!